ui/src/components/imquery/MatchSelector.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <div class="match-container" v-if="propertyType">
+    <Dropdown :options="['typeOf', 'instanceOf']" v-model:model-value="propertyType" />
```

Comment:
don't use v-on if not variable (e.g. options=['typeOf','instanceOf']

Reply:
not working on objects/arrays built dynamically like this one